ledger/sandbox/src/main/scala/com/digitalasset/platform/store/CompletionFromTransaction.scala

```diff
@@ -3,29 +3,68 @@
 
 package com.digitalasset.platform.store
 
+import java.time.Instant
+
 import com.daml.ledger.participant.state.v1.TransactionId
+import com.digitalasset.api.util.TimestampConversion.fromInstant
 import com.digitalasset.daml.lf.data.Ref
 import com.digitalasset.ledger.{ApplicationId, CommandId}
 import com.digitalasset.ledger.api.domain
-import com.digitalasset.ledger.api.domain.CompletionEvent
-import com.digitalasset.ledger.api.domain.CompletionEvent.{CommandAccepted, CommandRejected}
+import com.digitalasset.ledger.api.domain.RejectionReason
+import com.digitalasset.ledger.api.domain.RejectionReason.{
+  Disputed,
+  Inconsistent,
+  OutOfQuota,
+  PartyNotKnownOnLedger,
+  SubmitterCannotActViaParticipant,
+  TimedOut
+}
```

Comment:
Seems unnecessary to import these twice. (Or at all; I'd prefer prefixing them.)

Reply:
`scalafmt` puts newlines that can reduce the readability but I'll do this, if we don't like it we can always turn back, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4675/commits/7cedc01cfb3513928288f672123e578ed0d6f4a8